HACKING.md

```diff
@@ -41,50 +43,47 @@ To check if the watcher daemon is running, use `docker ps`. Then you can use `do
 
 ### Running the site with native python
 
-Since the site is basically a Django app, you can also run the site in the traditional way using [python 2.7](https://www.python.org/download/releases/2.7/) and [virtualenv](http://docs.python-guide.org/en/latest/dev/virtualenvs/):
+Since the site is basically a Flask app, you can also run the site in the traditional way using [python 3](https://docs.python.org/3/) and [venv](https://docs.python.org/3/library/venv.html?highlight=venv#module-venv):
 
 ``` bash
-virtualenv env
+python3 -m venv env
 source env/bin/activate
 pip install -r requirements.txt
-./entrypoint 0.0.0.0:8001
+./entrypoint 127.0.0.0:8001
```

Comment:
This needs to be either `./entrypoint 127.0.0.1:8001` or `./entrypoint 0.0.0.0:8001`. I'd recommend the latter just 'cos that's what we use in `./run`.